website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Model Timing Tab"
+id: "cloud-model-timing-tab"
+---
+
+### Overview
+Accessed via the "run detail" page in dbt Cloud, the model timing dashboard displays the model composition, order, and run time for every job run in dbt Cloud. The top 1% of model durations are automatically highlighted for quick reference.  This visualization is displayed after the run completes.
+
+This is a very visual way to explore your run. Longest running models *may* be ripe for further exploration -- which can lead to refactoring or reducing run cadence.
+
+Note that this is only available on multi-tenant Team and Enterprise accounts.
+
+<LoomVideo id="28a49a5c511c4063b4a3381cb81a03cf" />
```

Comment:
Loom vid is great. I might also include a screenshot of the viz right at the top of the page, to really grab attention. Only downside of Loom is it takes a few seconds to load/render.